docs/debug.md

```diff
@@ -0,0 +1,48 @@
+# CMake Debugging
+
+Starting with CMake 3.27, debugging CMake is now supported in CMake Tools!
+
+The following documentation will help you understand the various ways you can debug CMake.
+
+## Debugging from CMake Tools UI entry points
+
+The most common reason to debug CMake is to debug CMake Configuration! There are many ways that you can accomplish this:
+
+* Commands
+  * CMake: Configure with CMake Debugger
+  * CMake: Delete Cache and Reconfigure with CMake Debugger
+* Folder Explorer
+  * Right click on CMakeLists.txt context menu entry.
+* Project Outline
+  * Right click on CMakeLists.txt context menu entry.
+  * Expand the "..." in the project outline. There is an entry to use the Debugger!
+
+## Debugging from CMake Tools launch.json
+
+CMake Tools now provides a new debug type `cmake`!
```

Comment:
Here and above at the beginning of the document, I don't think we need to say "now". That implies a point-in-time comparison with an older version of the extension, which would make sense in an announcement like a blog post, but this is persistent documentation.

Reply:
That makes sense. Changing.